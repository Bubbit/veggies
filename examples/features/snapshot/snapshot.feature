@snapshot
Feature: Using snapshot definitions

    Scenario: Snapshot testing on an api
        Given I mock http call to forward request body for path /users/yaml
        And set request json body from file
        When I POST http://fake.io/users/yaml
        Then response body should match snapshot

    Scenario: Snapshot testing on cli
        When I run command echo test
        Then exit code should be 0
        And stdout output should match snapshot

    Scenario: Snapshot testing on files
        Given I set cwd to examples/features/snapshot/fixtures
        Then file file.yaml should match snapshot
