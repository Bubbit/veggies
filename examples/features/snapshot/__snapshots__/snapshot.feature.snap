

exports[`Setting json body from .yaml fixture file 1.1`] = `Object {
  "first_name": "Raphaël",
  "gender": "male",
  "id": 1,
  "last_name": "Benitte",
}`;

exports[`Snapshot testing on an api 1.1`] = `Object {
  "first_name": "Raphaël",
  "gender": "male",
  "id": 1,
  "last_name": "Benitte",
}`;

exports[`Snapshot testing on cli 1.1`] = `"test
"`;

exports[`Snapshot testing on files 1.1`] = `"id:         1
first_name: Raphaël
last_name:  Benitte
gender:     male
"`;
